job-tutorial/README.md

```diff
@@ -76,10 +76,17 @@ A json file describe detailed configuration required for a job submission. The d
     {
       "name":       String,
       "taskNumber": Integer,
+      "useTheSamePorts": Boolean,
       "cpuNumber":  Integer,
       "memoryMB":   Integer,
       "gpuNumber":  Integer,
-      "command":    String
+      "command":    String,
+      "portDefinitions": {
+        "label": {
+          "start": Integer,
+          "count": Integer
+        }
```

Comment:
RestServer have not supported it, we can leave the doc to yifan.

Reply:
comment them in model.

---
In reply to: [173127762](https://github.com/Microsoft/pai/pull/318#discussion_r173127762) [](ancestors = 173127762)